src/redux/slices/support/middleware.ts

```diff
@@ -0,0 +1,28 @@
+import { IHelpDataProps } from '@allTypes/reduxTypes/supportStateTypes'
+import API from '@axios/API'
+import { IError } from '@axios/authentication/authManagerTypes'
+import { AppDispatch } from '@redux/store'
+
+import slice from './slice'
+
+const { setError, setHelpLoading, setHelpStatus } = slice.actions
+
+const fetchHelpMessage = (message: IHelpDataProps) => async (dispatch: AppDispatch) => {
+  try {
+    dispatch(setHelpLoading(true))
+
+    const response = await API.support.sendHelpData(message)
+
+    dispatch(setHelpStatus(response.status))
```

Comment:
This should be boolean and in success case should be true

Reply:
done